bleak/backends/bluezdbus/scanner.py

```diff
@@ -245,3 +245,15 @@ def _handle_advertising_data(self, path: str, props: Device1) -> None:
         )
 
         self._callback(device, advertisement_data)
+
+    def _handle_device_removed(self, device_path: str) -> None:
+        """
+        Handles a device being removed from BlueZ.
+        """
+        try:
+            self._devices[device_path]
```

Comment:
```suggestion
            del self._devices[device_path]
```

Should this be `del`?

Reply:
yes, of course!